libs/designsystem/src/lib/components/tabs/tabs.component.spec.ts

```diff
@@ -17,37 +19,77 @@ describe('TabsComponent', () => {
-    });
-  });
+    describe('screensize phone', () => {
+      beforeAll(async () => {
+        await TestHelper.resizeTestWindow(TestHelper.screensize.phone);
+      });
 
-  describe('tabBarBottomHidden: false', () => {
-    it('should show tab bar', () => {
-      spectator.component.tabBarBottomHidden = false;
-      spectator.detectChanges();
+      describe('tabBarBottomHidden: true', () => {
+        beforeEach(() => {
+          spectator.component.tabBarBottomHidden = true;
+          spectator.detectChanges();
+        });
+        it('should hide tab bar when tabBarBottomHidden', async () => {
+          const ionTabBarElm = spectator.query('ion-tab-bar');
+          expect(ionTabBarElm).toHaveComputedStyle({ display: 'none' });
+        });
+        it('should NOT set page footer safe area override if tab bar bottom is hidden', async () => {
+          const ionTabBarElm = spectator.query('ion-tab-bar');
+          expect(ionTabBarElm).toHaveComputedStyle({
+            '--kirby-page-footer-safe-area-bottom': '',
+          });
+        });
+      });
 
-      const ionTabBarElm = spectator.query('ion-tab-bar');
-      expect(ionTabBarElm).not.toHaveComputedStyle({ display: 'none' });
+      describe('tabBarBottomHidden: false', () => {
+        beforeEach(() => {
+          spectator.component.tabBarBottomHidden = false;
+          spectator.detectChanges();
+        });
+        it('should show tab bar', () => {
+          const ionTabBarElm = spectator.query('ion-tab-bar');
+          expect(ionTabBarElm).not.toHaveComputedStyle({ display: 'none' });
+        });
+        it('should set footer safe area to 0 when tab bar is shown and in bottom', () => {
+          const ionTabBarElm = spectator.query('ion-tab-bar');
+          expect(ionTabBarElm).toHaveComputedStyle({
+            '--kirby-page-footer-safe-area-bottom': '0px',
+          });
+        });
+      });
     });
-    it('should set footer safe area to 0 when tab bar is shown and in bottom', () => {
-      spectator.component.tabBarBottomHidden = false;
-      spectator.detectChanges();
 
-      const ionTabBarElm = spectator.query('ion-tab-bar');
-      expect(ionTabBarElm).toHaveComputedStyle({ '--kirby-page-footer-safe-area-bottom': '' });
+    describe('screensize large', () => {
+      beforeEach(async () => {
+        await TestHelper.resizeTestWindow({ width: '1200px', height: '900px' });
+      });
+
+      describe('tabBarBottomHidden: true', () => {
+        it('should NOT hide tab bar when tabBarBottomHidden', async () => {
+          spectator.component.tabBarBottomHidden = true;
+          spectator.detectChanges();
+          const ionTabBarElm = spectator.query('ion-tab-bar');
+
+          expect(ionTabBarElm).not.toHaveComputedStyle({ display: 'none' });
+        });
+      });
+
+      describe('tabBarBottomHidden: false', () => {
+        it('should not set footer safe area', async () => {
```

Comment:
Align wording with other test:
```suggestion
        it('should NOT set page footer safe area override', async () => {
```